nerfstudio/model_components/ray_samplers.py

```diff
@@ -42,10 +42,10 @@ def __init__(
         self.num_samples = num_samples
 
     @abstractmethod
-    def generate_ray_samples(self) -> RaySamples:
+    def generate_ray_samples(self) -> Any:
```

Comment:
Why `Any`?

Reply:
Due to inconsistent types in overrided functions.